src/plugins/vis_type_timelion/server/routes/run.ts

```diff
@@ -88,6 +98,7 @@ export function runRoute(
           allowedGraphiteUrls: configManager.getGraphiteUrls(),
           esShardTimeout: configManager.getEsShardTimeout(),
           savedObjectsClient: context.core.savedObjects.client,
+          getIndexPatternsService: async () => indexPatternsService,
```

Comment:
nit: I see that code was copied from `TSVB`. But it that plugin we had a reason to return `indexPatternsService` in a `Promise`. For your case it doesn't have a lot of sense.

 I suggest you to remove line 91: `const indexPatternsService = await getIndexPatternsService();`

And here write:
```
{
    savedObjectsClient: context.core.savedObjects.client,
    getIndexPatternsService,
}

```